kernel.go

```diff
@@ -359,7 +372,7 @@ func handleExecuteRequest(ir *interp.Interp, receipt msgReceipt) error {
 
 	if executionErr == nil {
 		// if one or more value is image.Image, display it instead
-		vals = publishImages(vals, receipt)
+		vals = publishImages(vals, &receipt)
```

Comment:
If all of the functions simply render instead of display this is the place we get to take advantage of that. We need a type assertion but I think checking for a struct is fine. If we are trying to return a mimebundle we can get the rich output in the `Out[_]` in addition to the display calls.